sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ConflictResolutionPolicy.java

```diff
@@ -111,23 +112,47 @@ public static ConflictResolutionPolicy createLastWriterWinsPolicy(String conflic
      * required.
      *
      * <ul>
+     * <li>This method requires conflictResolutionStoredProcFullPath in format
+     * dbs/%s/colls/%s/sprocs/%s. User can also use equivalent method {@link #createCustomPolicy(String, String, String)}</li>
      * <li>In case the stored procedure fails or throws an exception,
      * the conflict resolution will default to registering conflicts in the conflicts feed</li>
      * <li>The user can provide the stored procedure @see {@link Resource#getId()} </li>
      * </ul>
      *
-     * @param conflictResolutionStoredProcName stored procedure to perform conflict resolution.
+     * @param conflictResolutionStoredProcFullPath stored procedure full path to perform conflict resolution.
      * @return ConflictResolutionPolicy.
      */
-    public static ConflictResolutionPolicy createCustomPolicy(String conflictResolutionStoredProcName) {
+    public static ConflictResolutionPolicy createCustomPolicy(String conflictResolutionStoredProcFullPath) {
```

Comment:
is this meant to be self link or name link?

Reply:
why do we need to provide this? if we have the other method:
    public static ConflictResolutionPolicy createCustomPolicy(String dbName, String containerName, String sprocName) {

Reply:
this will be breaking change,  createCustomPolicy(String conflictResolutionStoredProcName) is already public, we are just changing name of parameter for clarity , we can remove in V5.